src/Assets.php

```diff
@@ -114,6 +117,7 @@ public static function get_wc_block_data( $settings ) {
 				'isLargeCatalog'     => $product_counts->publish > 200,
 				'limitTags'          => $tag_count > 100,
 				'hasTags'            => $tag_count > 0,
+				'hasProducts'        => $product_counts->publish > 0,
```

Comment:
What would you think about naming this setting as `hasPublishedProducts`? I think it describes better its value.